app/src/main/java/com/asfoundation/wallet/billing/partners/ApkFyService.kt

```diff
@@ -0,0 +1,35 @@
+package com.asfoundation.wallet.billing.partners
+
+import android.content.Context
+import io.reactivex.Single
+import java.util.*
+import java.util.zip.ZipFile
+
+
+class ApkFyService(private val context: Context) {
```

Comment:
Can we change the name of this service to be according with the context of our project?